package.json

```diff
@@ -56,12 +56,15 @@
     "passport": "0.3.2",
     "passport-local": "1.0.0",
     "pug": "2.0.0-rc.1",
+    "scrypt-js": "2.0.3",
+    "semantic-ui-css": "2.2.10",
     "uuid": "3.0.1",
     "winston": "2.3.0"
   },
   "devDependencies": {
```

Comment:
Frontend Master course tip: Consider user package "rimraf" to delete dist folder before rebuilding it.
on scripts: "prebuild": "rimraf dist",